src/app/views/sprint/SprintPage.ts

```diff
@@ -0,0 +1,64 @@
+import { IWord } from '../../interfaces/IWord';
+import { WordService } from '../../services/WordService';
+import Renderable from '../Renderable';
+import { IResultData } from '../../interfaces/IResultData';
+import { SprintStartPage } from './SprintStartPage';
+import { SprintGameField } from './SprintGameField';
+import { IGameQuestionArray } from '../../interfaces/IGameQuestionArray';
+import { SprintStatisticPage } from './SprintStatisticPage';
+import { IGameAnswer } from '../../interfaces/IGameAnswer';
+import { IGameQuestion } from '../../interfaces/IGameQuestion';
+
+export class SprintPage extends Renderable {
+  constructor(group: number, page: number) {
+    super();
+    const startPage = new SprintStartPage(group - 1, page - 1);
+    const gameField = new SprintGameField();
+    startPage.onStartGame = (selectedGroup: number, selectedPage: number) => {
+      WordService.getWordsByGroupAndPage(selectedGroup, selectedPage)
+        .then((wordData) => {
+          const questionsData = {
+            questions: this.createQuestions(wordData.array),
+            currentQuestion: 0,
+          } as IGameQuestionArray;
+
+          console.log(questionsData);
```

Comment:
console.log

Reply:
fix in: https://github.com/thirdmadman/rslang/pull/34/commits/6d7be31790f68b7ba9119690718d0138ab10ec18